pages/_app.tsx

```diff
@@ -4,6 +4,7 @@ import { ThemeProvider } from "styled-components";
 import { hotjar } from "react-hotjar";
 import * as Sentry from "@sentry/browser";
 
+import { persistLinkReferrerCode } from "../lib/referrerCode";
```

Comment:
was this accidentally added back in?

Reply:
I didnt know if it was required and last night thought it might screw stuff up